pkg/kv/postgres/store.go

```diff
@@ -44,7 +44,7 @@ const (
 	// DefaultPartitions Changing the below value means repartitioning and probably a migration.
 	// Change it only if you really know what you're doing.
 	DefaultPartitions   = 100
-	DefaultScanPageSize = 1000
+	DefaultScanPageSize = 10000
```

Comment:
We can, but I think we should increase this number to 10K anyway. If we already scanning our store, why bring only 1000 items?

Reply:
This can have many unexpected results. Bringing 10k items when you only need the 1000 hurts performance. Can't we override it just for the single request?